mlflow/server/js/src/components/ExperimentView.css

```diff
@@ -194,23 +194,56 @@ span.error-message {
   text-overflow: ellipsis;
   white-space: nowrap;
   vertical-align: top;
+  height: 100%;
 }
 
-
 .ExperimentView .metric-param-container-cell {
   min-width: 280px;
 }
 
 .ExperimentView .metric-param-cell {
   display: inline-block;
   width: 250px;
-  padding: 2px;
+  padding: 4px;
 }
 
 .ExperimentView .metric-param-content {
-  padding: 4px;
+  padding-top: 0px;
 }
 
 .ExperimentView-expander:hover {
   cursor: pointer;
 }
+
+.ExperimentView .ReactVirtualized__Table__headerRow {
+  overflow: visible;
+  text-transform: inherit;
+  font-weight: inherit;
+}
+
+/* Workaround as per https://github.com/bvaughn/react-virtualized/issues/876#issuecomment-367029529 */
+.ExperimentView .ReactVirtualized__Table__rowColumn {
+  overflow: visible !important;
+}
+
+.ExperimentView .ReactVirtualized__Table__headerColumn {
+  color: rgb(136, 136, 136);
+}
+
+.ExperimentView .ReactVirtualized__Table .run-table-container {
+  padding: 8px;
+}
+
+.ExperimentView .ReactVirtualized__Table  .BaggedCell .run-table-container {
+  padding: 4px;
+}
+
+.flex-container {
```

Comment:
As discussed, could we think about a more descriptive name for this? Perhaps even more specific than the generally true "flex-vertical-container"